code/game/objects/items/RCD.dm

```diff
@@ -315,7 +356,6 @@ RLD
 		"Medical" = get_airlock_image(/obj/machinery/door/airlock/medical),
 		"Research" = get_airlock_image(/obj/machinery/door/airlock/research),
 		"Freezer" = get_airlock_image(/obj/machinery/door/airlock/freezer),
-		"Science" = get_airlock_image(/obj/machinery/door/airlock/science),
```

Comment:
?

Reply:
I din't cherrypick this file due to alot of errors but i will readd the deleted science airlock lines.